Aktualizacia web-core a nutna uprava configu

jankrcal opened the pull request:
Tento PR dělá změny vynucené z web-core (související s podporou podcastu).

Pro slovenský web to v důsledku neznamená žádné změny v chování webu (zatím tedy stále není možné odkazovat na český podcast 2050). Kdyby byl o to zájem, není problém v budoucnu dodělat.

jankrcal committed:
Aktualizacia web-core a nutna uprava configu

jankrcal commented:
@mgrabovsky Můžeš na to prosím kouknout? (nespěchá)

Uděláš si prosím opět podobnou změnu ve web-en sám nebo bys ocenil, kdybych to udělal já?

mgrabovsky reviewed (approved):
Vyzerá dobre. Len som posunul web-core na najnovší. O web-en sa postarám.

mgrabovsky committed:
Oprava komentářů

Pro  skryté komentáře v textu je třeba používat tag `{% comment %}`. Při
použití HTML `<!-- -->` uvnitř dochází k expanzi tagů, což v případě
`include` může rozbít build.

mgrabovsky commented:
@jankrcal Chybku, co rozbíjela build, jsem opravil. Pošlu ještě patch do web-core, který přidá check na správné místo. Jinak je to za mě OK.
